### PR TITLE
Update: Korean translation

### PR DESCRIPTION
**Pull request by telk5093**
There could be compile error, if so, please let me know.
You may squash merge this ;)

### CODE REVIEW

File: src/lang/extra/korean.txt [hunk 1]
```diff
@@ -1291,9 +1294,16 @@ STR_STATION_VIEW_HISTORY_BUTTON                                 :{BLACK}화물
...
 STR_DEPARTURES_DEPARTURES                                       :{BLACK}{TINY_FONT}발
 STR_DEPARTURES_ARRIVALS                                         :{BLACK}{TINY_FONT}착
-STR_DEPARTURES_VIA_BUTTON                                       :{BLACK}경유
+STR_DEPARTURES_BOTH_COMBINED                                    :혼합
+STR_DEPARTURES_BOTH_SEPARATE                                    :분리
+STR_DEPARTURES_DEPARTURE_MODE_TOOLTIP                           :{BLACK}출발이나 도착을 보여줄지 설정합니다
+STR_DEPARTURES_TIMES_BUTTON                                     :{BLACK}시각
+STR_DEPARTURES_TIMES_TOOLTIP                                    :{BLACK}시간표 상 목적지 도착 시각을 보여줍니다
+STR_DEPARTURES_EMPTY_BUTTON                                     :{BLACK}공차
+STR_DEPARTURES_EMPTY_TOOLTIP                                    :{BLACK}공차로 수송하거나, 화물을 싣고 내리지 않는 정차 역을 포함합니다.STR_DEPARTURES_VIA_BUTTON                                       :{BLACK}경유
```
Comment (JGRennison) [inline src/lang/extra/korean.txt:1306]: There seems to be two lines combined together here?

**Comment by JGRennison** [comment]
The compiler error should be fixed now, that was a problem in my code

File: src/lang/extra/korean.txt (hunk 1 above)
Comment (telk5093) [reply src/lang/extra/korean.txt]: Didn't noticed, thanks for notice

**Comment by telk5093** [comment]
Thanks for notice! (I don't care about compiler error :P)

**Comment by JGRennison** [comment]
Thanks, that's merged now.

**Pull request merged**